packages/hping3/build.sh

```diff
@@ -0,0 +1,11 @@
+TERMUX_PKG_HOMEPAGE=http://www.hping.org
+TERMUX_PKG_DESCRIPTION="hping is a command-line oriented TCP/IP packet assembler/analyzer."
+TERMUX_PKG_VERSION=3.0.0
+TERMUX_PKG_BUILD_IN_SRC="yes"
+TERMUX_PKG_DEPENDS="libandroid-shmem, libpcap"
+TERMUX_PKG_SHA256=f1684e568b6d00ba096e1c77b18be4500ca2b8d2715bb2059aff22b8b8717eff
+TERMUX_PKG_SRCURL=https://hax4us.github.io/hping/hping3-${TERMUX_PKG_VERSION}.tar.gz
+TERMUX_PKG_EXTRA_CONFIGURE_ARGS="--no-tcl"
+termux_step_pre_configure {
+  sudo apt-get install -y libpcap-dev
```

Comment:
i'm curious, how does linking against the build machine's `libpcap` help the build?

Reply:
It is just missing the header files. So by installing the libpcap-dev package you get those files.

Reply:
Btw, have you tried it on your own build? The Travis CI build failed because of my mistake yesterday. You can test it once again to see what happens.

Reply:
the problem is the headers from `sudo apt-get install ...` aren't associated with the termux package `libpcap`, those headers are from the ubuntu version of `libpcap`. `TERMUX_PKG_DEPENDS` basically gets/makes/installs/etc the headers needed

Reply:
But I tried without apt-get install .....
Same pcap.h not found...